crates/rust-analyzer/src/global_state.rs

```diff
@@ -365,6 +412,54 @@ impl GlobalState {
 
         self.analysis_host.apply_change(change);
 
+        let config_change = {
+            let mut change = ConfigChange::default();
+            let snap = self.analysis_host.analysis();
```

Comment:
```suggestion
            let snap = self.analysis_host.raw_database();
```
no need to create a snapshot here